config/initializers/open_library.rb

```diff
@@ -0,0 +1 @@
+OPEN_LIBRARY = YAML.load_file("#{::Rails.root}/config/open_library.yml")[::Rails.env]
```

Comment:
👀  please read this article https://blog.saeloun.com/2019/10/10/rails-6-adds-support-for-multi-environment-credentials.html#in-rails-6 and change it